src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -0,0 +1,71 @@
+import { getCLS, getFID, getLCP, getFCP, getTTFB } from 'web-vitals';
+
+
+const jsonData = {
+    'page_view_id': '',
+    'fid': 0,
+    'cls': 0,
+    'lcp': 0,
+    'fcp': 0,
+    'ttfb': 0,
+}
+
+export const coreVitals = (): void =>
+{
+    type coreVitalsArgs = {
+        name: string;
+        value: number;
+    };
+
+
+const getValue = ({ name, value }: coreVitalsArgs): void => {
+    switch(name){
+        case('FCP'):
+            console.log(`FCP: ${  value}`);
```

Comment:
You probably want to as ESLint disable for console log to avoid getting build errors